local/owr_local_media_source.c

```diff
@@ -86,6 +98,10 @@ G_DEFINE_TYPE_WITH_CODE(OwrLocalMediaSource, owr_local_media_source, OWR_TYPE_ME
 struct _OwrLocalMediaSourcePrivate {
     gint device_index;
     OwrMessageOriginBusSet *message_origin_bus_set;
+
+    /* Volume control for audio sources; tmp_volume is for before source_volume gets created */
+    GstElement *source_volume;
+    double tmp_volume;
```

Comment:
It is not only the initial volume, but the volume any time source_volume doesn't exist. Maybe covering that case isn't necessary.